### PR TITLE
Put the CMK enable feature behind a feature flag

### PR DESCRIPTION
**Pull request by cdavid**
The CMK enable feature is not yet available in production environment, so adding it behind a feature flag.

### CODE REVIEW

**Review by arishta-yb — APPROVED** [review]

File: cmd/util/feature_flags.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ const (
 	NODE_OP            FeatureFlag = "NODE_OPS"
 	TOOLS              FeatureFlag = "TOOLS"
 	AZURE_CIDR_ALLOWED FeatureFlag = "AZURE_CIDR_ALLOWED"
+	CLUSTER_CMK_UPDATE FeatureFlag = "CLUSTER_CMK_UPDATE"
```
Comment (kc-yb) [inline cmd/util/feature_flags.go:33]: Do we have YBM CLI UTs to test this?

Comment (cdavid) [reply cmd/util/feature_flags.go:33]: @kc-yb - what unit tests do you have in mind? Is it for this specific feature flag or the feature flag functionality in general?

Comment (kc-yb) [reply cmd/util/feature_flags.go:33]: In general, and for this as well - ie, turning the feature flag does not show the command

Comment (kc-yb) [reply cmd/util/feature_flags.go:33]: If you have doc on YBM CLI UT appreciate if you can share it

**Pull request merged**